interface-definitions/include/firewall/global-options.xml.i

```diff
@@ -44,6 +44,25 @@
       </properties>
       <defaultValue>disable</defaultValue>
     </leafNode>
+    <node name="apply-for-bridge">
```

Comment:
I don't like this option name. First, it's ungrammatical: _apply_ is used with _for_ only in its intransitive sense ("apply for a job"), when it's transitive it should be "apply $something _to_ $somethingElse". Second, it's not very obvious. Maybe `apply-to-bridged-traffic` — kinda wordy, but it's immediately obvious what the option does.

Alternatively: `filter-bridged-traffic`.